app/src/test/java/ar/com/wolox/android/bootstrap/base/BaseViewModelTest.kt

```diff
@@ -1,38 +0,0 @@
-package ar.com.wolox.android.bootstrap.base
```

Comment:
I think this file must stay. If not, you need to create more boilerplate in every test class you make.